app/templates/base.html

```diff
@@ -70,16 +70,16 @@ <h1 id="error-title" class="card-title">{% block error_title %}Error{% endblock
 
         <div class="col col-sm-8 col-md-6 offset-md-3">
             <p>
-                <a href="https://sadilar.org" title="{% trans 'External link'%}">
-                    {% trans "Developed by SADiLaR" %}
+                <a href="{% url 'about' %}" title="{% trans 'About LwimiLinks'%}">
+                    {% trans "About LwimiLinks" %}
                 </a>
             </p>
             <div class="d-md-flex justify-content-center">
                 <div class="px-2">
                     <a href="{% url 'legal_notices' %}" class="btn btn-link p-0">{% trans "Legal Notices" %}</a>
                 </div>
                 <div class="px-2">
-                    <a href="{% url 'contact' %}" class="btn btn-link p-0">{% trans "Contact Us" %}</a>
+                    <a href="{% url 'about' %}#contact" class="btn btn-link p-0">{% trans "Contact Us" %}</a>
                 </div>
```

Comment:
Two links to the same page is subobtimal. I propose we just drop this div entirely.

Reply:
I feel like these links appeal to different users - it is not 100% clear if the user is unfamiliar that the contact and about pages are one and the same. They might want to ctrl+F "contact" and find some info